code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -90,6 +96,14 @@
 				return
 			accesses -= get_region_accesses(region)
 			. = TRUE
+		if("passedName")
+			var/new_name = trim("[params["passedName"]]", 30)
+			passed_name = new_name
+			. = TRUE
+		if("passedCycleId")
+			var/new_cycle_id = trim(params["passedCycleId"], 30)
+			passed_cycle_id = new_cycle_id
+			. = TRUE
```

Comment:
Sanitize these when they come in, rather than when they are used.

Reply:
Done in c6de418.

passedname and passedcycleId both seem to work after sanitizing. Let me know if I fucked up the format